### PR TITLE
Add support for AMDGPU target gfx1100

### PR DESCRIPTION
**Pull request by mhalk**
These minor changes allow the use of gfx1100 AMDGPUs, i.e. RX 7900 XT(X).
To the best of my knowledge, using 'ctest' and 'cgsolve' no regressions occurred.

### CODE REVIEW

File: cmake/kokkos_arch.cmake [hunk 1]
```diff
@@ -91,10 +91,11 @@ IF(Kokkos_ENABLE_HIP OR Kokkos_ENABLE_OPENMPTARGET OR Kokkos_ENABLE_UNSUPPORTED_
...
 
-KOKKOS_ARCH_OPTION(VEGA900         GPU  "AMD GPU MI25 GFX900"      "KOKKOS_SHOW_HIP_ARCHS")
-KOKKOS_ARCH_OPTION(VEGA906         GPU  "AMD GPU MI50/MI60 GFX906" "KOKKOS_SHOW_HIP_ARCHS")
-KOKKOS_ARCH_OPTION(VEGA908         GPU  "AMD GPU MI100 GFX908"     "KOKKOS_SHOW_HIP_ARCHS")
-KOKKOS_ARCH_OPTION(VEGA90A         GPU  "AMD GPU MI200 GFX90A"     "KOKKOS_SHOW_HIP_ARCHS")
+KOKKOS_ARCH_OPTION(VEGA900         GPU  "AMD GPU MI25 GFX900"           "KOKKOS_SHOW_HIP_ARCHS")
+KOKKOS_ARCH_OPTION(VEGA906         GPU  "AMD GPU MI50/MI60 GFX906"      "KOKKOS_SHOW_HIP_ARCHS")
+KOKKOS_ARCH_OPTION(VEGA908         GPU  "AMD GPU MI100 GFX908"          "KOKKOS_SHOW_HIP_ARCHS")
+KOKKOS_ARCH_OPTION(VEGA90A         GPU  "AMD GPU MI200 GFX90A"          "KOKKOS_SHOW_HIP_ARCHS")
+KOKKOS_ARCH_OPTION(VEGA1100        GPU  "AMD GPU RX 7900 XT(X) GFX1100" "KOKKOS_SHOW_HIP_ARCHS")
```
Comment (dalg24) [inline cmake/kokkos_arch.cmake:98]: Isn't gfx1100 "Navi" instead of "Vega"?

**Review by dalg24 — COMMENTED** [review]
Make sure you target the develop branch.
Did you mean to close?

File: cmake/kokkos_arch.cmake (hunk 1 above)
Comment (mhalk) [reply cmake/kokkos_arch.cmake:98]: You're absolutely right, very blatant oversight TBH.

**Comment by mhalk** [comment]
Yes, I did mean to close since I did not want to use up your time with this PR, as I realized that multiple things are messed up.
I'd like to fix things up, but I want to double-check that it's working and my current dev environment is not based around develop.
Really sorry here!

**Pull request closed without merging**